### DISCUSSION TITLE
Ignores Closes word in PR

### DISCUSSION THREAD

**Discussion by yaroslavyaroslav** [community/community#38459] [Mobile] [answered]
So the `Closes #565` text within PR description is ignored. I mean it gets highlighted like it doesn't, but when pr is merged issue keeps opened. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.87.0 (82263682)
Device: iPhone 12 mini (iOS 16.1)
</details>

**Comment by aconsuegra** [comment, marked as answer]
Hi @yaroslavyaroslav, thanks for reporting the issue. I haven't been able to reproduce the issue, so I might need some help from you.

Do you see an icon like this one in the list of pull requests for the ones that include the `Closes ...` keyword?
<img width="396" alt="Screen Shot 2022-11-09 at 14 10 26" src="https://user-images.githubusercontent.com/1814882/200839286-2461295b-ff25-4d55-8a41-e8d4bc58bed0.png">

I just want to make sure that the pull request and the issue are properly linked for you.

**Reply by yaroslavyaroslav** [reply]
Let's make an experiment with this [pr](https://github.com/skywinder/web3swift/pull/658). 

I'll merge it whenever it'll be ready from an iPhone, and check will close the issue. 

![image](https://user-images.githubusercontent.com/16612247/200867241-996c389b-affe-4073-94c2-466ae2efb3c0.png)

**Reply by aconsuegra** [reply]
Ok, that looks good and issue and pull request are properly linked. Merging the pull request should close the issue...

**Reply by yaroslavyaroslav** [reply]
It works well that time. Seems that time when it was failed I've added it somewhere in the comments bellow rather then into body of the PR.
